esm-samples/jsapi-create-react-app/README.md

```diff
@@ -29,9 +29,12 @@ If you run into issues with babel helpers injected into ArcGIS API for JavaScrip
 {
   "browserslist": {
     "production": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
+      "last 1 Chrome version",
+      "last 1 Firefox version",
+      "last 2 Edge major versions",
+      "last 2 Safari major versions",
+      "last 2 iOS major versions",
+      "Firefox ESR"
     ],
     "development": [
       "last 1 chrome version",
```

Comment:
Thanks @benelan, let's mirror those settings for both dev and prod.

That's also brings it in-line with what we use here: https://github.com/Esri/jsapi-resources/blob/master/esm-samples/jsapi-angular-cli/.browserslistrc

Reply:
Should I update the react sample's `package.json` too?

Reply:
Yessir, might as well since you have to manually update it anyway.